packages/core/README.md

```diff
@@ -294,12 +295,21 @@ The `clear-text` reporter supports three additional config options:
 
 The `dashboard` reporter sends a report to https://dashboard.stryker-mutator.io, enabling you to add a mutation score badge to your readme, as well as hosting your html report on the dashboard. It uses the [dashboard.*](#dashboard) configuration options. See [the Stryker handbook](https://github.com/stryker-mutator/stryker-handbook/blob/master/dashboard.md) for more info.
 
-<a name="sandbox.fileHeaders"></a>
-### `sandbox.fileHeaders` [`object`]
+<a name="sandbox.ignorePatterns"></a>
+### `sandbox.ignorePatterns` [`string[]`]
```

Comment:
So this is like a "master switch"? Wouldn't that be confusing? What does this mean:

```js
{
  sandbox: {
    ignorePatterns: "**/*.js",
    removeComments: "**/*.js"
  }
```

Reply:
> So this is like a "master switch"?

yes

My idea was that it should be possible to ignore all files in `__tests__` for example, even though some files in `__tests__` would match a pattern such as `**/*.{js,ts}` or `**/!(*.spec,*.test,*Spec).{js,ts}`.

But yeah, this idea would probably be too confusing. Back to the drawing board.